### ISSUE TITLE
Snapshot and Stream Errors FFmpeg exited with code -4058

### ISSUE THREAD

**Issue by ElanX** [KhaosT/homebridge-camera-ffmpeg#341]
**I am getting the following errors on all my cameras I just added to Home:**
> [8/17/2019, 2:46:28 AM] [Camera-ffmpeg] An error occurs while making stream request
> [8/17/2019, 2:46:28 AM] [Camera-ffmpeg] ERROR: FFmpeg exited with code -4058
> [8/17/2019, 2:46:38 AM] [Camera-ffmpeg] Start streaming video from Front Door with 1280x720@299kBit
> [8/17/2019, 2:46:38 AM] [Camera-ffmpeg] An error occurs while making stream request
> [8/17/2019, 2:46:38 AM] [Camera-ffmpeg] ERROR: FFmpeg exited with code -4058
> [8/17/2019, 2:46:42 AM] [Camera-ffmpeg] Snapshot from Garage at 480x270
> [8/17/2019, 2:46:42 AM] [Camera-ffmpeg] An error occurs while making snapshot request
> [8/17/2019, 2:46:42 AM] [Camera-ffmpeg] Snapshot from Front Door at 480x270

**My configs look like this:**
>             "name": "Front Door",
>                     "videoConfig": {
>                     "source": "-re -i http://admin:XXX@18.0.0.8/h264/frontdoor/temp.ts",
>                   "stillImageSource": "-i http://admin:XXX@18.0.0.8/image/frontdoor?q=75",
>                     "maxStreams": 2,
>                     "maxWidth": 1280,
>                     "maxHeight": 1024,
>                     "maxFPS": 15

**I am able to bring up the still images and the streams in my browser no problem. I've tried messing with lots of settings, but no luck, yet. I saw one post say a 4058 error is a path issue. I added a couple FFmpeg folders to my path, but not sure I added the right one.**

**Comment by ElanX** [comment]
Have now tried 8 cameras of different brands and protocols, and all have the same issues, so I'm pretty sure it's something with my config or FFmpeg. Any ideas @KhaosT ?

**Comment by Bad2dabone** [comment]
ffmpeg not installed follow instuction below once installed all will work
https://github.com/adaptlearning/adapt_authoring/wiki/Installing-FFmpeg

**Comment by ElanX** [comment]
> ffmpeg not installed follow instuction below once installed all will work
> https://github.com/adaptlearning/adapt_authoring/wiki/Installing-FFmpeg

I had FFMpeg installed. Double checked it was in path by running it from a command prompt in root. Realized my mistake though. I ran Homebridge as administrator and it works now! Maybe that's obvious, but didn't need to for any of my other plugins. Maybe that should be added to instructions?

Only thing is Homebridge froze after a few minutes. And after a restart, one camera stopped working for 5 mins and then came back. But these are separate issues. Closing this one!
